### PR TITLE
Order notes column

### LINKED ISSUE #437: Show order notes in the model table
following #436 (following #242)

### PR DESCRIPTION
**Pull request by melinath**
Resolved #437. Set the order notes column to display in the order table, and also moved the notes form to the front of the order detail page so it's easier to find & use.

### CODE REVIEW

**Comment by melinath** [comment]
Went ahead and fully resolved #242 as well as part of this PR.

File: brambling/models.py [hunk 1]
```diff
@@ -1459,6 +1459,9 @@ class Attendee(AbstractNamedModel):
 
     custom_data = GenericRelation('CustomFormEntry', content_type_field='related_ct', object_id_field='related_id')
 
+    # Admin-only data
```
Comment (harrislapiroff) [inline brambling/models.py]: I'm gonna change this to say "Organizer-only data". We're admins. Our customers are organizers.

Comment (melinath) [inline brambling/models.py]: okay

**Comment by harrislapiroff** [comment]
The UI here is pretty wonky (confusing to distinguish between attendee notes and order notes; filling out multiple forms and pressing one save button only saves one form and loses all the other entered data) but I think I'm going to merge it because I'm pretty sure the best fix is to have the fields in the tables, which will require more significant changes.

**Comment by harrislapiroff** [comment]
Actually maybe I'll clean it up a little.

**Comment by harrislapiroff** [comment]
No wait, I can't. Not without restructuring the form. Eh. Too much. We'll fix it later.

**Pull request merged**